src/rust/ide/view/src/documentation/style.css

```diff
@@ -319,7 +319,7 @@
 }
 
 .Synopsis {
-    margin-top: 15px;
```

Comment:
This works well with `pure` documentation, but will break `AST` doc, as far as i remember there is doc title which  then can have margin-bottom: 15px